stubs/src/lib.rs

```diff
@@ -2,88 +2,25 @@
 // https://github.com/tokio-rs/prost/issues/661#issuecomment-1156606409
 #![allow(clippy::derive_partial_eq_without_eq)]
 
+pub mod atmosphere;
+pub mod coalition;
+pub mod common;
+pub mod controller;
+pub mod custom;
+pub mod group;
+pub mod hook;
+pub mod mission;
+pub mod net;
+pub mod timer;
+pub mod trigger;
+pub mod unit;
 mod utils;
-
-pub mod atmosphere {
-    pub mod v0 {
-        tonic::include_proto!("dcs.atmosphere.v0");
-    }
-}
-
-pub mod coalition {
-    pub mod v0 {
-        tonic::include_proto!("dcs.coalition.v0");
-    }
-}
-
-pub mod common {
-    pub mod v0 {
-        tonic::include_proto!("dcs.common.v0");
-    }
-}
-
-pub mod controller {
-    pub mod v0 {
-        tonic::include_proto!("dcs.controller.v0");
-    }
-}
-
-pub mod custom {
-    pub mod v0 {
-        tonic::include_proto!("dcs.custom.v0");
-    }
-}
-
-pub mod group {
-    pub mod v0 {
-        tonic::include_proto!("dcs.group.v0");
-    }
-}
-
-pub mod hook {
-    pub mod v0 {
-        tonic::include_proto!("dcs.hook.v0");
-    }
-}
-
-pub mod mission {
-    pub mod v0 {
-        tonic::include_proto!("dcs.mission.v0");
-    }
-}
-
-pub mod net {
-    pub mod v0 {
-        tonic::include_proto!("dcs.net.v0");
-    }
-}
-
-pub mod timer {
-    pub mod v0 {
-        tonic::include_proto!("dcs.timer.v0");
-    }
-}
-
-pub mod trigger {
-    pub mod v0 {
-        tonic::include_proto!("dcs.trigger.v0");
-    }
-}
-
-pub mod unit {
-    pub mod v0 {
-        tonic::include_proto!("dcs.unit.v0");
-    }
-}
-
-pub mod world {
-    pub mod v0 {
-        tonic::include_proto!("dcs.world.v0");
-    }
-}
+pub mod world;
```

Comment:
I've moved all those `mod`s into a separate file. This improved the incremental compilation a lot when making changes to one of them.

Reply:
Very nice